src/targets/browser/index.jsx

```diff
@@ -50,9 +52,7 @@ const schema = {
   }
 }
 
-document.addEventListener('DOMContentLoaded', () => {
-  if (handleOAuthResponse()) return
-
+const setupAppContext = memoize(() => {
```

Comment:
What is the goal of this memoization? Is it to avoid multiple cozy-client instanciation because of the hot reload?

Reply:
Yes.

Reply:
I added a comment to clarify.